<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="29659" started-at="2021-12-07T04:40:49Z" finished-at="2021-12-07T04:41:19Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="29659" started-at="2021-12-07T04:40:49Z" finished-at="2021-12-07T04:41:19Z">
      <class name="datashare.TestNG_DataStore_ThreadLocal_01">
        <test-method status="PASS" signature="createBooking()[pri:1, instance:datashare.TestNG_DataStore_ThreadLocal_01@c8e4bb0]" name="createBooking" duration-ms="9553" started-at="2021-12-07T04:40:49Z" finished-at="2021-12-07T04:40:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createBooking -->
        <test-method status="FAIL" signature="retrieveBooking()[pri:2, instance:datashare.TestNG_DataStore_ThreadLocal_01@c8e4bb0]" name="retrieveBooking" duration-ms="6" started-at="2021-12-07T04:40:59Z" finished-at="2021-12-07T04:40:59Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at datashare.TestNG_DataStore_ThreadLocal_01.retrieveBooking(TestNG_DataStore_ThreadLocal_01.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- retrieveBooking -->
      </class> <!-- datashare.TestNG_DataStore_ThreadLocal_01 -->
      <class name="datashare.TestNG_DataStore_ThreadLocal_02">
        <test-method status="PASS" signature="createBooking()[pri:1, instance:datashare.TestNG_DataStore_ThreadLocal_02@62e136d3]" name="createBooking" duration-ms="9556" started-at="2021-12-07T04:40:49Z" finished-at="2021-12-07T04:40:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createBooking -->
        <test-method status="FAIL" signature="retrieveBooking()[pri:2, instance:datashare.TestNG_DataStore_ThreadLocal_02@62e136d3]" name="retrieveBooking" duration-ms="20012" started-at="2021-12-07T04:40:59Z" finished-at="2021-12-07T04:41:19Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at datashare.TestNG_DataStore_ThreadLocal_02.retrieveBooking(TestNG_DataStore_ThreadLocal_02.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- retrieveBooking -->
      </class> <!-- datashare.TestNG_DataStore_ThreadLocal_02 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
